sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -28,7 +28,9 @@ import {
   BlobStartCopyFromURLResponse,
   BlobAbortCopyFromURLResponse,
   BlobCopyFromURLResponse,
-  BlobSetTierResponse
+  BlobSetTierResponse,
+  ContainerCpkScopeInfo,
+  CpkScopeInfo,
```

Comment:
Minor: At first glance on CpkScopeInfo and ContainerCpkScopeInfo, it looks like CpkScopeInfo > ContainerCpkScopeInfo, and might be confused to user. Please just double check this aligns with other SDKs, and passes API review.

Reply:
In the API review, they agree on renaming `ContainerCpkScopeInfo` to `ContainerEncryptionScope`.